### ISSUE TITLE
NTR: Spleen biopsy: necrosis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9119]
New term request Spleen biopsy: necrosis (elthat_100424153159): Splenic infarction  UMLS: 1096363

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000232
- label: Splenic necrosis
- parents: [Abnormal spleen morphology (HP:0025408)](https://hpo.jax.org/app/browse/term/HP:0025408)
- pmid: PMID:30949357
- def: Cell dealth (necrosis) located in the spleen.
- comment: CT of the abdomen can be used to investigate splenic necrosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9119
- synonyms:

**Comment by MickeySegal** [comment]
At some point we added https://hpo.jax.org/app/browse/term/HP:0034336 to our term.  Should these remain separate or be combined?
